ui/apps/platform/src/Containers/ComplianceEnhanced/Coverage/CoveragePage.tsx

```diff
@@ -42,8 +43,7 @@ function CoverageContent() {
     }
 
     if (!isLoading && scanConfigProfilesResponse.totalCount === 0) {
-        // TODO: Add a message for when there are no profiles
-        return <div>No profiles, create a scan schedule</div>;
+        return <CoverageEmptyState />;
```

Comment:
Being pedantic here. what I observed on dogfooding today is that creation of a schedule does not trigger the condition. Only when scan results are available the empty state message disappears. I suggest we adjust the condition or—simpler—the empty state text.

Reply:
The empty state should only appear when no schedules have been created. If a schedule is created, we display an empty table while waiting for the first set of results.

![Screenshot 2024-06-25 at 12 29 39 PM](https://github.com/stackrox/stackrox/assets/61400697/c0b5cc04-9dc1-4ded-ba53-b8114ac8e965)